.github/workflows/keep-alive.yaml

```diff
@@ -0,0 +1,17 @@
+name: Keep
+on:
+  schedule:
+    - cron: "0 9 * * 3"  # every wednesday at 9am
```

Comment:
Do we want to align with the other cron job?

https://github.com/conda-forge/docker-images/blob/d948295308cc06cc6d147612d4c5bd645522c42c/.github/workflows/ci.yaml#L8-L9

Reply:
Sure, why not.
```suggestion
  schedule:
    - cron: "0 6 * * SUN"  # Once weekly on Sunday @ 0600 UTC 
```